spikingjelly/activation_based/cuda_kernel/auto_cuda/neuron_kernel/plif.py

```
@@ -657,34 +657,22 @@ def multistep_plif(
-            v_threshold,
-            v_reset_value,
-            soft_reset,
-            detach_reset,
-            decay,
-            decay_input,
-            sg_cupy_id,
-        )
-        return s_seq, v_seq
+            return s_seq, v_seq
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)